index.Rmd

```diff
@@ -28,7 +28,11 @@ a:visited {color: #91170a;}
 * Use control flow for efficient function writing
 
 # Basic *R* knowledge useful for Geo-Scripting
-Scripting with *R* to handle spatial problems requires a core set of basic *R* skills. To prepare you well for the coming weeks, we've summarized what we think are important skills for geoscripting below.
+Scripting with *R* to handle spatial problems requires a core set of basic *R* skills. To prepare you well for the coming weeks, we've summarized what we think are important skills for geoscripting below. 
+
+```{block type="alert alert-info"}
+**Note** that this tutorial uses the basic programming structure with lists, strings, and more. For a refresher on these elements click [here](https://geoscripting-wur.github.io/RPythonBasics/).
```

Comment:
Wanted to put a reference to the R and Python Basics tutorial in case students skipped it. I am not 100% sure how hyperlinks work in Rmd, I tried to look at other scripts for reference, but my links could have errors...

On another note, it could be nice to have an additional section on getting started with RStudio. Going over the RStudio structure, the console, ... It is probably that some students have never used any IDE before.